protocols/bgp/packet/path_attributes.go

```diff
@@ -37,6 +56,12 @@ func decodePathAttrs(buf *bytes.Buffer, tpal uint16, opt *DecodeOptions) (*PathA
 		}
 	}
 
+	if haveNextHop || haveOrigin || haveASPath {
+		if !haveNextHop || !haveOrigin || !haveASPath {
+			return nil, fmt.Errorf("A mandatory path attribute is missing")
```

Comment:
Lets ease debugging and print which mandatory path attribute is missing.

Reply:
Done.